src/libslic3r/GCode/FanMover.cpp

```diff
@@ -83,6 +87,12 @@ int16_t get_fan_speed(const std::string &line, GCodeFlavor flavor) {
         if (flavor == (gcfMach3) || flavor == (gcfMachinekit)) {
             return (int16_t)get_axis_value(line, 'P');
         } else {
+            // BBS: for Bambu machine, we both use M106 P1 and M106 to indicate the part cooling fan
```

Comment:
Since we support not just Bambu machines, ref to: https://github.com/SoftFever/OrcaSlicer/wiki/Auxiliary-fan and https://github.com/SoftFever/OrcaSlicer/wiki/air-filtration.
Would it be better to update the comment here to reduce confusion?

Reply:
well.. it's nice to know this part of the code came from bambuslicer, no ? also it's not an untrue statement.. it may also be true for other printers as well, but this one explicitly says what the case is for that one.

Reply:
> well.. it's nice to know this part of the code came from bambuslicer, no ? also it's not an untrue statement.. it may also be true for other printers as well, but this one explicitly says what the case is for that one.

This is not true. This feature was ported into OrcaSlicer from SuperSlicer by @scottmudge at PR https://github.com/SoftFever/OrcaSlicer/pull/910

> it may also be true for other printers as well, but this one explicitly says what the case is for that one.

I'm unsure of what point you're trying to make. This feature is not specific to Bambu machines; Orca supports the use of M106 P1 and M106 commands for the part cooling fan on all printers, which is the basis for my previous comment.

Reply:
oh, well nevermind then.

Reply:
https://github.com/SoftFever/OrcaSlicer/blob/ae31f4fb1fd42eab674008145c187b76faad7560/src/libslic3r/GCode/GCodeProcessor.cpp#L3934

FYI I copied this from here.

Reply:
The problem is that BambuLab uses P1 as the part cooling fan, not P0, which is what this check really meant for (because we don't check for P0).